rebasehelper/tests/conftest.py

```diff
@@ -27,9 +27,18 @@
 
 import pytest  # type: ignore
 
+from rebasehelper.specfile import SpecFile
+
 
 TESTS_DIR: str = os.path.dirname(__file__)
 TEST_FILES_DIR: str = os.path.join(TESTS_DIR, 'testing_files')
+SPEC_FILE: str = 'test.spec'
+
+
+@pytest.fixture
+def spec_object(workdir):  # pylint: disable=redefined-outer-name
+    sf = SpecFile(SPEC_FILE, workdir)
+    return sf
```

Comment:
I think it would be more convenient if the fixture would copy the specfile to `workdir`, to avoid the need to put it in `TEST_FILES`.